tools/integration_tests/operations/operations_test.go

```diff
@@ -74,6 +74,11 @@ const NumberOfObjectsInDirThreeInCreateThreeLevelDirTest = 1
 const PrefixFileInDirThreeInCreateThreeLevelDirTest = "fileInDirThreeInCreateThreeLevelDirTest"
 const FileInDirThreeInCreateThreeLevelDirTest = "fileInDirThreeInCreateThreeLevelDirTest1"
 const ContentInFileInDirThreeInCreateThreeLevelDirTest = "Hello world!!"
+const DestEmptyCopyDirectory = "destEmptyCopyDirectory"
```

Comment:
Can we group the variables based on operation? E.g.

// Read
Read related variable...

// Write
Write related variable...

// Copy
Copy related variable...